gossip/src/cluster_info.rs

```diff
@@ -2688,10 +2698,9 @@ impl ClusterInfo {
                                     table_size,
                                 );
                             }
-                            GossipError::DuplicateNodeInstance => {
+                            GossipError::DuplicateNodeInstance(msg) => {
                                 error!(
-                                    "duplicate running instances of the same validator node: {}",
-                                    self.id()
+                                    "duplicate running instances of the same validator node: {msg}"
```

Comment:
If we're still looking to improve reporting, I could see bubbling up the duplicate IP address as being useful. In the "intentional" takeover case, someone probably knows that IP. But, if it is something like we were debugging where we had a forgotten node running with the same keys, the IP in the message would make it very clear. What do you think ?

Reply:
> What do you think ? Might be over-analyzing this PR 😅

Good idea, I'll add the IP to the message.